deploy/charts/venafi-kubernetes-agent/Chart.yaml

```diff
@@ -3,5 +3,5 @@ name: venafi-kubernetes-agent
 description: |-
   The Venafi Kubernetes Agent connects your Kubernetes or Openshift cluster to the Venafi Control Plane.
 type: application
-version: 1.0.0
-appVersion: "v1.0.0"
+version: 1.1.0-alpha.0
```

Comment:
These versions should just remain 0.0.0 in the repo. Makefile modules will overwrite them normally.

Reply:
See https://github.com/jetstack/venafi-enhanced-issuer/blob/main/deploy/charts/venafi-enhanced-issuer/Chart.yaml

Reply:
Thanks. I've fixed this.